instagram_own_version/settings.py

```diff
@@ -62,8 +63,12 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "corsheaders.middleware.CorsMiddleware",
```

Comment:
Let's move CORS related stuff it to another PR

Reply:
Removed